README.md

```diff
@@ -48,3 +48,68 @@ Any use of third-party trademarks or logos are subject to those third-party's po
 ## libazureinit
 
 For common library used by this reference implementation, please refer to [libazureinit](https://github.com/Azure/azure-init/tree/main/libazureinit/).
+
+## Tracing Logic Overview
```

Comment:
maybe this should be its own doc under /doc/?

Reply:
Agreed -- I will make a /doc folder and add it there instead of in the ReadME.